bundles/net.tourbook/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -11,7 +11,7 @@ org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enablePreviewFeatures=disabled
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
 org.eclipse.jdt.core.compiler.problem.reportPreviewFeatures=warning
-org.eclipse.jdt.core.compiler.release=disabled
+org.eclipse.jdt.core.compiler.release=enabled
```

Comment:
> What is the reason to change this setting?
@wolfgang-ch 
I didn't change those settings intentionally, I don't understand yet where this modification is coming from !

Reply:
Depending ony your action within Eclipse it set states in files which are difficult to detect.

For me, this change would be no problem when I could understand what it means, especially when building the product then I don't want to have issues which are difficult to detect and finally it was caused e.g. by this setting change.

The only solution to find such modifications is to check **EVERY** modification before committing it. If you would have done this, then the encoding issue where easily detected by yourself and had saved time for others to review/comment it.

Normally, before I do a commit, I check any modifications, this also helps e.g. to remove commented code. Checking every modification takes a bit of time but in the end it can save much more time when the code must be repaired.

Reply:
>Normally, before I do a commit, I check any modifications, this also helps e.g. to remove commented code. Checking every modification takes a bit of time but in the end it can save much more time when the code must be repaired.

It's a good recommendation, however, because the Pull requests are not done against main, then the branches are typically out dated and more files show as "modified" when only a few files were actually modified which makes easy to miss modifications like these.

Reply:
Example, in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/260/files), I only modified the WorldWind jars yet 116 files show as modified !!!
So, yes it's good to check any modifications but to check 116 files when I only modified 10+ files, no thank you

![image](https://user-images.githubusercontent.com/25235665/108887905-0bfd7100-75c8-11eb-8b92-ae27e4ac6759.png)

Reply:
> > Normally, before I do a commit, I check any modifications, this also helps e.g. to remove commented code. Checking every modification takes a bit of time but in the end it can save much more time when the code must be repaired.
> 
> It's a good recommendation, however, because the Pull requests are not done against main, then the target branches are typically out dated and sometimes more files show as "modified" when only a few files were actually modified which makes easy to miss modifications like these.

When I merge branches and committing it, then I do not check the modifications in this commit, I check only modifications which I've done manually in the Eclipse IDE and normally there are only a few modified files.